website_attribute_set/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2011 Kencove (http://www.kencove.com).
+# @author Mohamed Alkobrosli <malkobrosly@kencove.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Website Attribute Set",
+    "version": "18.0.1.0.0",
+    "category": "Website/Website",
+    "license": "AGPL-3",
+    "author": "Kencove, Odoo Community Association (OCA)",
+    "maintainers": ["kobros-tech"],
+    "website": "https://github.com/OCA/odoo-pim",
+    "depends": [
+        "attribute_set",
+        "product_attribute_set",
+        "pim",
+        "website",
+        "website_sale",
```

Comment:
```suggestion
    "depends": [
        "pim",
        "website_sale",
```

`pim` already includes `attribute_set` and `product_attribute_set`, similarly `website_sale` also includes `website`

Reply:
I am inheriting methods and xml records from these modules!

Reply:
I am inheriting from each module there